webapp-django/crashstats/crashstats/models.py

```diff
@@ -694,6 +693,7 @@ class ProcessedCrash(SocorroMiddleware):
 
 
 class UnredactedCrash(ProcessedCrash):
+    implementation = SimplifiedCrashData
```

Comment:
Subclassing and not specifying the `implementation` messes with the way the implementations get mocked out in the tests. We should fix the tests to do a better job of patching, but it was easier to do this for now.